lib/hooks/index.js

```diff
@@ -136,33 +132,33 @@ function activate(agent) {
     // time that we need to go and patch pro-actively.
```

Comment:
The indentation is wrong here. If you're in this code, please cleanup the indentation too.